js/client/modules/@arangodb/air/pregel-example-graphs.js

```diff
@@ -206,7 +206,49 @@ function createCircle(graphName, numberOfVertices, numberOfShards) {
   return { vname: vname, ename: ename };
 }
 
-function createLineGraph(graphName, numberOfVertices, numberOfShards) {
+function writeToLeaf(obj, path) {
+  path = path.split('.');
+  let res = obj;
+
+  for (var i=0; i < path.length; i++) {
+    if (res[path[i]]) {
+      res = res[path[i]];
+    } else {
+      res[path[i]] = {};
+    }
+  }
+}
+
+function objectInsertHelper(arr, valueToInsert) {
+  let obj = {};
+  arr.forEach((key) => {
+    if (typeof key === "string") {
+      obj[key] = valueToInsert;
+    } else {
+      // array expected (toplevel -> n-level)
+      _.set(obj, key, valueToInsert);
+
+      //let pathAsArray = key;
+      //let pathAsString = "";
+      /*for (let pos = 0; pos < pathAsArray.length; pos++) {
+        if (pos === 0) {
+          // first iteration
+          pathAsString = pathAsArray[pos];
+        } else if (pos < (pathAsArray.length - 1)) {
+          // middle iteration (object buildup)
+          pathAsString += '.' + pathAsArray[pos];
+        } else {
+          // last step (also insert value i here)
+          _.set(obj, pathAsString, )
+        }
+      }*/
+    }
```

Comment:
What is this and why is it commented out?

Reply:
Ah, this code I wanted to use before I've found that there is the underscore set method :) 
Will remove it completely now.